docs/_posts/2020-05-22-seminar-minutes-2020-10.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Bot Friday 2020年第10次活动纪要
+author: ggsonic
+categories: events
+tags:
+  - minutes
+header:
+  teaser: /assets/2020/seminar-10/group-photo.jpg
+---
+
+<< Bot5 2020年第9次活动纪要: [Bot5 Season 2020 Episode 9](https://bot5.club/events/seminar-minutes-2020-09) <<
+
+## Bot Friday Season 2020 Episode 10
+
+- 时间：2020年5月22日（周五）19:00 - 21:00（2小时）
+- 地点：Zoom在线视频会议
+- 轮值主席：
+    1. Chair: 郭成凯
+    1. Vice-Chair: 张凯
+
+![attendees]({{ '/assets/2020/seminar-10/group-photo.jpg' | relative_url }})
+
+## Attendees
+
+1. 王创峰，前去哪儿网算法工程师
+1. 郭成凯，gpt2 transformer-xl 爱好者
+1. [李卓桓](/people/huan/)：A Chatbot Builder
+1. 张卓然，CMU ECE本科生，在做Octopedia留学咨询chatbot
+1. 李云军，基于Chatbot团队协作的研究与探索
+1. 张凯， Chatopera Inc.聊天机器人创业者。
+
+新人：
+
+1. 张卓然，CMU ECE本科生，在做Octopedia留学咨询chatbot
+
+## Talks
+
+### 视频
+
+<div class="video-container" style="
+    position: relative;
+    padding-bottom:56.25%;
+    padding-top:30px;
+    height:0;
+    overflow:hidden;
+">
+  <iframe width="560" height="315"
+    src="https://www.youtube.com/embed/buurLyAD8Tw"
```

Comment:
Can we embed the video twice for each of our talkers, and set the `src` of YouTube link at exactly the start time at they start presenting?

We can merge this now, however it would be better if we can let the reader watch from the start of each presentation with only one click in our blog.